devtools-ci-index.yaml

```diff
@@ -4052,6 +4065,13 @@
             ci_cmd: '/bin/bash cico_build_test.sh'
             timeout: '20m'
             discarder_days: 30
+        - '{ci_project}-{git_repo}-build-master':
```

Comment:
object with secret that you've defined is not used by any `job-template` so your job will not get that secret injected

Reply:
So I would recommend you to create new `job-template` based on `job_template_build_defaults` and add `wrapper` that will use secret that you've created

Reply:
I have been trying to figure out how to do this by going through various internal docs and devguide. Is there a doc for this or am I supposed to guess based on `job_template_build_defaults` ?

Reply:
I was going through various internal docs and devguide to figure out how to do this. Is there a doc about this or am I supposed to guess it from `job_template_build_defaults`?

Reply:
here is jjb docs: https://docs.openstack.org/infra/jenkins-job-builder/index.html